src/components/TimeCardPage/TimeTableRow.tsx

```diff
@@ -12,29 +12,50 @@ import {TIMEZONE} from 'src/constants';
 
 import {CellType} from './types' 
 import { TimeEntry } from './CellTypes/TimeEntry';
-import {Duration} from './CellTypes/HoursCell' 
+import {Duration} from './CellTypes/HoursCell'  
 import {DateCell} from './CellTypes/DateCell'; 
 import { TypeCell } from './CellTypes/CellType';
 import { CommentCell } from './CellTypes/CommentCell';
 import {RowSchema} from '../../schemas/RowSchema'; 
+import ApiClient from 'src/components/Auth/apiClient'
+
+import * as updateSchemas from 'src/schemas/backend/UpdateTimesheet'
+import apiClient from 'src/components/Auth/apiClient';
 
 interface RowProps {
     row: RowSchema; 
     prevDate: number; 
     onRowChange: Function; 
+    TimesheetID: number; 
 } 
 
+
+
+
 function Row(props: RowProps) { 
   
     const [fields,setFields] = useState<undefined | RowSchema>(undefined); 
 
     const updateField = (key, value) => {
+        
         const newFields = {
             ...fields, 
             [key]: value 
         }
+
         setFields(newFields); 
         props.onRowChange(newFields); 
+        //Send a request to update the db on this item being changed 
+        ApiClient.updateTimesheet(updateSchemas.TimesheetUpdateRequest.parse({
```

Comment:
Is this going to be called every time a field is changed, or only when the user clicks 'save'? either way is probably fine for now, but it might be a little excessive if it's every time something is changed. if that's the case, maybe just leave a TODO for adding in a timed auto-save instead?

Reply:
At some point we had discussed it and I believe the consensus was to call it everytime mainly since our current setup handles a single change at a time - uploading at saves / an auto-save time would require us to track these changes and then upload a list of them at a time or call it several times at that point removing from the queue of changes - should probably discuss this in a meeting / plan a new ticket for this if we want to change how this works as at the moment the endpoint just accepts a single update at a time.